frontend/src/Components/Api/algorithmDefaultParams.js

```diff
@@ -28,7 +28,9 @@ export const loadForecastDefaultParams = {
     res_l2_smooth: true,
     week_day: true,
     publicl2_daily_use: 0.33,
-    mixed_batteries: "None",
+    small_batt: 0,
```

Comment:
heads up on changing these variable names if the snake case was replaced with camel case, as per Jonathan's comment

Reply:
Most of these were already snake case when we got the app. I'll go ahead and update these also.

Reply:
fixed